processing/data_collection/gazette/pipelines.py

```diff
@@ -103,6 +107,18 @@ def txt_source_text(self, item):
         ) as f:
             return f.read()
 
+    def img_source_text(self, item):
+        """
+        Gets the text from image files
+        """
+        img_path = os.path.join(FILES_STORE, item["files"][0]["path"])
+        text_path = img_path + ".txt"
+        command = f"java -jar /tika-app.jar --text {img_path}"
```

Comment:
I'm afraid that does not work. At least, after run the spider locally I cannot see the txt file with the gazette content As far as I know, Apache Tika is not able to does OCR. Did you check if you are able to read the text from the scraped items in the txt files?

If we cannot convert the images files into text. Keep only the image file is better than have an empty text file.

Reply:
Oooh, I forgot to add something in the PR description, my bad.

In one of the commits' description, I mentioned it was not working and thought I should bring it up here that I found in the [docs](https://tika.apache.org/1.24.1/formats.html#Image_formats) that OCR could be done using the [Tesseract Parser](https://tika.apache.org/1.24.1/api/org/apache/tika/parser/ocr/TesseractOCRParser.html) but I couldn't get it to work. Since you added Tika to the project, maybe you knew how to do it :)

Also, there is the possibility of trying to use [Tesseract](https://github.com/tesseract-ocr/tesseract) itself as a standalone package.

How do you think we should approach this? Try to get Tika to work with the Tesseract Parser; try the Tesseract standalone package; ditch everything in favor of dealing with this later (this spider sucks your soul, come on, images?!); any other ideas?

Reply:
Hmmm, I'm not sure if the tesseract is available in the command line app. Let's keep only the image now. At least until we found a better solution. Update the pipeline item in your PR avoiding the text extraction when it's an image.

Reply:
Done. I put a warning log for image extractions